src/commands/server/start.ts

```diff
@@ -402,6 +413,10 @@ export default class Start extends Command {
       await postInstallTasks.run(ctx)
       this.log('Command server:start has completed successfully.')
     } catch (err) {
+      const isEmptyDir = await this.isDirEmpty(ctx.directory)
+      if (isEmptyDir) {
+        this.error(`${err}\nThere are no any logs available for the current installation`)
+      }
       this.error(`${err}\nInstallation failed, check logs in '${ctx.directory}'`)
```

Comment:
`check logs in '${ctx.directory}` doesn't make much sense if there are no any logs available.